pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -79,20 +72,20 @@ export class SystemInfomationCard extends React.Component {
                         name = fields.board_name;
                     }
                     if (!vendor || !name)
-                        self.setState({ hardwareText: undefined });
+                        this.setState({ model: null });
                     else
-                        self.setState({ hardwareText: vendor + " " + name });
+                        this.setState({ model: vendor + " " + name });
 
-                    self.setState({ assetTagText: fields.product_serial || fields.chassis_serial || undefined });
+                    this.setState({ assetTag: fields.product_serial || fields.chassis_serial });
                 })
                 .catch(ex => {
                     // try DeviceTree
-                    machine_info.devicetree_info()
-                            .then(fields => self.setState({ assetTagText: fields.serial, hardwareText: fields.model || undefined }))
+                    machine_info.devicetree_info() // not-covered: coverage only runs on x86_64
+                            .then(fields => this.setState({ assetTag: fields.serial, model: fields.model }))
                             .catch(dmiex => {
-                                console.debug("couldn't read dmi info: " + ex);
+                                console.debug("couldn't read dmi info: " + ex.toString());
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19119-20230721-060543-f5c512c9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)